lib/coinbase/address/wallet_address.rb

```diff
@@ -144,5 +177,13 @@ def ensure_sufficient_balance!(amount, asset_id)
 
       raise InsufficientFundsError.new(amount, current_balance)
     end
+
+    def complete_staking_operation(amount, asset_id, action, mode: :default, options: {})
+      op = StakingOperation.create(amount, network_id, asset_id, id, wallet_id, action, mode, options)
+      op.transactions.each do |transaction|
+        transaction.sign(@key)
```

Comment:
nodejs has checks around skipping signing if already signed - is that something we should do here as well ?

Reply:
No as we dont do it for other actions